### ISSUE TITLE
Enable a11y eslint rule

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/blog-constructor#167]
- Enable a11y rules in eslint config: https://github.com/gravity-ui/eslint-config?tab=readme-ov-file#a11y and disable if the issue came from rule cannot be fixed
- Provide an issue for each disabled rule with instructions on how to fix it

**Comment by NikitaCG** [comment]
create issue to fix cases - https://github.com/gravity-ui/blog-constructor/issues/171
